app.php

```diff
@@ -407,6 +407,7 @@ public function load_admin_scripts($hook) {
 					'backbone',
 					'backbone.trackit',
 					'_ca_condition-groups',
+					'vaa_view_admin_as_script',
```

Comment:
Haha I understand. What kind of conflicts did you have? Maybe there is an other solution for this issue :)